.github/workflows/test.yml

```diff
@@ -918,7 +918,7 @@ jobs:
             './tfm/cypress/e2e/journeys/case-amendments/facility-end-date/feature-flag-enabled/*.spec.js',
             './tfm/cypress/e2e/journeys/case-amendments/in-progress/*.spec.js',
             './tfm/cypress/e2e/journeys/case-amendments/request-when-previously-submitted/*.spec.js',
-            './tfm/cypress/e2e/journeys/case-amendments/show-amendment-button/*.spec.js',
+            './tfm/cypress/e2e/journeys/case-amendments/show-amendment-button/**/*.spec.js',
```

Comment:
Realised some tests were not being picked up on the pipeline, fixed that here

Reply:
This was causing some issues, so reverted this fix and will do this in a separate PR (theses tests are not linked to my PR otherwise)